contribute/style-guides/styling.md

```diff
@@ -113,5 +113,5 @@ const ComponentA: React.FC<Props> = ({ className }) => {
   );
 
   return <div className={finalClassName}>As red as you can ge</div>;
-};
+}
```

Comment:
For style guide maybe we can include two example components, first with `function Component() { }` declaration and the second with `const Component = () => { }` declaration given we have both forms throughout codebase? (i.e. both are acceptable ways of defining a component)